### ISSUE TITLE
FR: Support 3D images in AMP

### ISSUE THREAD

**Issue by aghassemi** [ampproject/amphtml#18892]
@mixtur Do you know whether `amp-3d-gltf` can support images with depth map that iOS produces in portrait mode directly? (https://www.facebook.com/hamilton.dave/posts/10157851313039838)

If not, do you know how/if they can be converted to `gltf` format? (ideally client-side so we can support `<amp-3d-gltf src="3dimage>` or maybe even add a new component `<amp-3d-photo src=3dimage">`

**Comment by donmccurdy** [comment]
I'm not precisely sure what iOS produces, but it's not a valid glTF file. It should be possible to convert them to glTF (I've seen an article on this somewhere, but can't find it now...). That does sound like a non-trivial change in scope for `amp-3d-gltf`. If you're going to load the image and depth map client-side, it would be simpler just to show them directly in three.js, without converting to glTF at runtime.

On the other hand, a simple GUI converter that packs the image and depthmap into a single `.glb` asset would also be pretty easy, similar to https://glb-packer.glitch.me. Do you have an example of what iOS produces?

**Comment by jwold** [comment]
1. I tried this on my Mac with Chrome, and saw the Facebook image supplied; that it moves around a bit when you hover the mouse. Is this all we're intending, or are we thinking more like a 360° option? 
2. I tried this on my iPhone X with Safari on Facebook.com and nothing seems to happen. Should something be happening here?
3. Is the intention to just be able to offer this 3D image as part of AMP? If so, would we just grab this from a portrait mode picture that I've taken? 


--

Beyond those initial questions I'd love to know if there's further UX feedback you're hoping for, or expecting. Thanks!
